hfjobs/commands/run.py

```diff
@@ -38,20 +38,28 @@ def register_subcommand(parser: _SubParsersAction) -> None:
             "--env-file", type=str, help="Read in a file of environment variables."
         )
         run_parser.add_argument(
-            "--flavor", type=str, help="Flavor for the hardware, as in HF Spaces.", default="cpu-basic"
+            "--flavor",
+            type=str,
+            help="Flavor for the hardware, as in HF Spaces.",
+            default="cpu-basic",
         )
         run_parser.add_argument(
-            "--timeout", type=str, help="Max duration: int/float with s (seconds, default), m (minutes), h (hours) or d (days)."
+            "--timeout",
+            type=str,
+            help="Max duration: int/float with s (seconds, default), m (minutes), h (hours) or d (days).",
         )
         run_parser.add_argument(
-            "-d", "--detach", action="store_true", help="Run the Job in the background and print the Job ID.", 
+            "-d",
+            "--detach",
+            action="store_true",
+            help="Run the Job in the background and print the Job ID.",
         )
         run_parser.add_argument(
-            "--token", type=str, help="A User Access Token generated from https://huggingface.co/settings/tokens"
-        )
-        run_parser.add_argument(
-            "command", nargs="...", help="The command to run."
+            "--token",
+            type=str,
+            help="A User Access Token generated from https://huggingface.co/settings/tokens",
         )
+        run_parser.add_argument("command", nargs="+", help="The command to run.")
```

Comment:
just noticed this change that breaks passing arguments that start with a `-`, will revert if you don't mind @drbh 

```
hfjbos run ubuntu ls -al
```

Reply:
Ah great catch sorry about that, yes let's revert and I'll open an improved PR soon. Thanks!

Reply:
reverted and released already :p